src/libs/portfolio/getOnchainBalances.ts

```diff
@@ -176,14 +176,23 @@ export async function getNFTs(
       : null
 
     const token = mapToken(beforeToken)
+    const [receiving, sending]: [bigint[], bigint[]] = [[], []]
+    beforeToken[2].forEach((oldCollectible: bigint) => {
```

Comment:
here, it would be better to use named properties from mapToken (i.e. `token.collectibles`), instead of indices, as it will improve the readability